README.md

```diff
@@ -19,18 +19,42 @@ as `abc` version `X.Y.Z+dune`.
 
 ## Port a new package/version to dune on `opam-overlays`
 
-The typical workflow to port an existing package to dune and publish it on
-`opam-overlays` would be the following:
-
-1. Create a fork of the upstream package `abc`
-2. Port it to dune on a specific branch `dune-universe-vX.Y.Z`, where `X.Y.Z`
-   corresponds to the upstream version you're dunifying. Make sure the opam file
-   has been properly updated and that dune is now part of the package `depends`
-   field and the `dev-repo` and `homepage` point to your fork. It is also
-   recommended to remove the `doc` field to ease the rest of the release
-   process.
-3. Tag it with `X.Y.Z+dune`.
-4. Use `dune-release` to release your port to this repo as described in the
+The typical workflow to port an existing package `pkg`, version `X.Y.Z` to dune
+and publish it on `opam-overlays` would be the following:
+
+1. Create a fork of the upstream package `pkg` repo on dune-universe
+2. Create a branch `dune-universe-vX.Y.Z` pointing to the upstream `X.Y.Z`
+   release tag
+3. Work on the actual port. Make sure the opam file has been properly updated
+   and that dune is now part of the package `depends` field. The build
+   instructions should also be updated to use dune. To ease the release process,
+   the `dev-repo` and `homepage` should point to the dune-universe fork. It is
+   also recommended to remove the `doc` field for similar reasons.
+   Once it is ready, create a PR to merge your port into the
+   `dune-universe-vX.Y.Z` branch of the dune-universe fork.
+4. Once the PR is reviewed and merged, tag the head commit of the
+   `dune-universe-v.X.Y.Z` with `X.Y.Z+dune`.
```

Comment:
Maybe better use `dune-release tag` for it, since it creates an annotated tag which will work with releasing manually but also with `dune-release`.

Reply:
`dune-release tag --tag` you mean? Yeah, either works for me, I usually do the annotated tag myself but you're right this needs to be clarified.

Reply:
I added the instructions to create the tag, both with and without `dune-release` to offer an escape hatch if dune-release struggles there for some reason.